ui/lib/core/addon/helpers/replication-mode-description.js

```diff
@@ -5,7 +5,7 @@
 
 import { helper as buildHelper } from '@ember/component/helper';
 
-const REPLICATION_MODE_DESCRIPTIONS = {
+export const REPLICATION_MODE_DESCRIPTIONS = {
```

Comment:
It looks like this variable isn't actually used anywhere (even though it's exported from the engine). I think instead we should remove the export from `app/helpers/replication-mode-description`